golangci-lint | `Explictly` is a misspelling of `Explicitly` (misspell)

From a recent linting failure:

```
##[error]config/merge_incomplete_configs_test.go:226:9: `Explictly` is a misspelling of `Explicitly` (misspell)
	t.Log("Explictly setting AppMetadata fields to pass validation checks")
	       ^
##[error]config/merge_incomplete_configs_test.go:353:9: `Explictly` is a misspelling of `Explicitly` (misspell)
	t.Log("Explictly setting AppMetadata fields to pass validation checks")
	       ^
```

--- a/config/merge_incomplete_configs_test.go
+++ b/config/merge_incomplete_configs_test.go
@@ -222,7 +222,7 @@ func TestMergeConfigUsingIncompleteConfigObjects(t *testing.T) {
 
 	envConfig := Config{}
 
-	t.Log("Explictly setting AppMetadata fields to pass validation checks")
+	t.Log("Explicitly setting AppMetadata fields to pass validation checks")
 	envConfig.AppName = baseConfig.GetAppName()
 	envConfig.AppVersion = baseConfig.GetAppVersion()
 	envConfig.AppURL = baseConfig.GetAppURL()
@@ -349,7 +349,7 @@ func TestMergeConfigUsingIncompleteConfigObjects(t *testing.T) {
 
 	flagsConfig := Config{}
 
-	t.Log("Explictly setting AppMetadata fields to pass validation checks")
+	t.Log("Explicitly setting AppMetadata fields to pass validation checks")
 	flagsConfig.AppName = baseConfig.GetAppName()
 	flagsConfig.AppVersion = baseConfig.GetAppVersion()
 	flagsConfig.AppURL = baseConfig.GetAppURL()

